Lib/importlib/_abc.py

```diff
@@ -1,11 +1,16 @@
 """Subset of importlib.abc used to reduce importlib.util imports."""
 from . import _bootstrap
 import abc
+import warnings
 
 
 class Loader(metaclass=abc.ABCMeta):
 
     """Abstract base class for import loaders."""
+    def __init__(self):
+        warnings.warn(f"Loader is deprecated.",
+                DeprecationWarning, stacklevel=2)
```

Comment:
I'd indent the arguments to be on the same level:

```suggestion
        warnings.warn(f"Loader is deprecated.",
                      DeprecationWarning, stacklevel=2)
```

Or you can even format it on a single line